crates/runtime/benches/queries/tpcds/q35.sql

```diff
@@ -10,8 +10,8 @@ select
   cd_dep_employed_count,
   count(*) cnt2,
   max(cd_dep_employed_count),
-  stddev_samp(cd_dep_employed_count),
```

Comment:
This is unrelated change, additional update as per https://github.com/spiceai/spiceai/pull/3069